### PR TITLE
fix invalid json state test fillers

### PR DESCRIPTION
**Pull request by cdetrio**
Did not try filling with these fixes, but all filler files in src/GeneralStateTests are now valid json.

### CODE REVIEW

**Comment by winsvega** [comment]
so now all of this comments will end up getting into the final tests
wouldn't it be confusing ?

**Comment by cdetrio** [comment]
Yea we don't want that. Perhaps `testeth` can ignore keys that begin with `//`?

File: src/GeneralStateTestsFiller/stZeroKnowledge/pairingTestFiller.json [hunk 1]
```diff
@@ -130,7 +130,7 @@
 		"[1] vk.B + proof.B + negateG1(proof.Bp) + P2",
 		"[2] proof.C + vk.C + negateG1(proof.Cp) + P2",
 		"[3] proof.K + vk.gamma + negateG1(addG1(vkx, addG1(proof.A, proof.C))) + vk.gammaBeta2 + negateG1(vk.gammaBeta1) + proof.B",
-		"[4] addG1(vkx, proof.A) + proof.B + negateG1(proof.H) + vk.Z +	negateG1(proof.C) + P2",
+		"[4] addG1(vkx, proof.A) + proof.B + negateG1(proof.H) + vk.Z + negateG1(proof.C) + P2",
 		"[5] proof.A + vk.A + proof.Ap + P2"
 	    ],
             "data" : [
```
Comment (pirapira) [inline src/GeneralStateTestsFiller/stZeroKnowledge/pairingTestFiller.json:136]: To me, the field `"data"` seems to be duplicated.  Is this still a valid JSON?

**Comment by pirapira** [comment]
I created an issue about removing `\\` fields https://github.com/ethereum/cpp-ethereum/issues/4309

File: src/GeneralStateTestsFiller/stZeroKnowledge/pairingTestFiller.json (hunk 1 above)
Comment (pirapira) [reply src/GeneralStateTestsFiller/stZeroKnowledge/pairingTestFiller.json:136]: I opened an issue about this https://github.com/ethereum/tests/issues/252

Comment (cdetrio) [reply src/GeneralStateTestsFiller/stZeroKnowledge/pairingTestFiller.json:136]: technically duplicate keys are valid JSON. I had to look it up: "The names within an object SHOULD be unique." So duplicate keys are not recommended but not forbidden.

Luckily the schema validator we're using only validates the last key/value when there are duplicates.

**Pull request merged**